### PR TITLE
My assignment 3

### PR DESCRIPTION
**Pull request by koushikalakhamraju**

### CODE REVIEW

**Comment by koushikalakhamraju** [comment]
Screenshot (78)

File: src/Assignment-3/assignment3.html [hunk 1]
```diff
@@ -12,7 +12,24 @@
...
 
 	<!-- Start your code from here -->
-	
+	<div class="container-1">
+		<div class="col-1">
+			<h1>your best value proposition</h1>
+			<p>It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout.  </p>
+			<button type="button">start free trial!</button>
+		</div>
+		<div id="render-wiki">
```
Comment (praveendhakade) [inline src/Assignment-3/assignment3.html]: iframe should have the id #render-wiki here.

File: src/Assignment-3/assignment3.html [hunk 2]
```diff
@@ -12,7 +12,24 @@
...
+			<iframe src="https://www.wikipedia.org/" style="width: 100%; height: 200%;"></iframe>
+		</div>
+	</div>
+	<div class="container-2">
+			<h1>your best value proposition</h1>
+			<p>It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout. of a page when looking at its layout. </p>
+			<button type="button">start trial!</button>
+	</div>
+	<div id="render-dictionary">
```
Comment (praveendhakade) [inline src/Assignment-3/assignment3.html]: here also iframe should have the id.

**Review by praveendhakade — CHANGES_REQUESTED** [review]

**Comment by koushikalakhamraju** [comment]
updated it with above modifications

**Review by praveendhakade — APPROVED** [review]
Great!

**Pull request merged**